content/07.pathogenesis.md

```diff
@@ -372,32 +364,33 @@ In contrast, a model using the prevalence of antibodies among residents of Wuhan
 An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
-They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
+The argument was thus made that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated asymptomatic individuals.
 While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.
 
 #### Estimating the Fatality Rate
 
 Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
 The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
-One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
-However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
+One commonly reported metric is the case fatality rate (CFR), which compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
+However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
 Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
-Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854], and this estimate was also supported by a repeated cross-sectional serosurvey conducted in New York City that revealed an estimated IFR of 0.97% [@doi:10.1038/s41586-020-2912-6].
+Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780].
+This was also supported by a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].
```

Comment:
I had learned in high school that "this" can't be a noun on its own -- there either needs to be a predicate noun ("This is an excellent study") or a noun directly after it (e.g., "This finding was also supported...").  However, I cannot for the life of me find/remember the name of this rule!!! That's why I originally had "This estimate" and had stuck a lot of nouns in while giving people feedback, but now I'm worrying I made up this rule...

Reply:
I've heard similar guidelines because it can lead to an [unclear antecedent](https://iconlogic.blogs.com/weblog/2010/11/grammar-workshop-can-i-start-a-sentence-with-this.html).

Reply:
```suggestion
A similar prevalence estimate was identified through a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].
Examination of serosurvey-based estimates of IFR identified convergence on a global IFR  estimate of 0.60% (95% CI 0.42%–0.77%) [@doi:10.1016/j.ijid.2020.09.1464].
```